src/test/java/runners/TestNGRunnerTest.java

```diff
@@ -0,0 +1,36 @@
+package runners;
+
+import com.ui.LoginReportPortalPage;
+import components.ui.BaseTest;
+import components.ui.LoginReportPortalTest;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+public class TestNGRunnerTest extends LoginReportPortalTest {
+
+    WebDriver driver;
+    LoginReportPortalPage loginReportPortalPage;
```

Comment:
I see that this object initialized in @AutrtTest what is confusing. You can move all objects that will be used in tests to the BaseTest class and initialize them there and use through all tests as you need.